bot/discord/commands/votemute.js

```diff
@@ -0,0 +1,54 @@
+const ms = require('ms');
+const Discord = require('discord.js');
+const mutesData = new require('quick.db').table("muteData");
+
+exports.run = async(client, message, args) => {
+  let modlog = message.guild.channels.cache.find(channel => channel.id === config.DiscordBot.modLogs);
+  if (!message.member.roles.cache.find(r => r.id === "898041754564046869")) return message.reply("Sorry, but it looks like you're too much of a boomer to run this command.");
+  if (args.length < 2) return message.channel.send('', { embed: new Discord.MessageEmbed().setColor(0x00A2E8).setDescription(`Correct usage ${config.DiscordBot.Prefix}votemute <@user|userID> [Time : 5m] [Reason : unspecified]`).setFooter('<required> [optional]')});
+  const allowed_channels = ["898041849783148585", "898041865616650240"];
+  if (allowed_channels.indexOf(client.channels.cache.get('id')) == -1) return; // Only lounge and dono lounge
```

Comment:
dude this is the most retarded part of the code yet. What if the issue is in another channel? you will need to go into a different completely random channel just to mute them. also how tf would this work?

Reply:
it would be so much easier to just do
```js
if(!allowed_channels.includes(message.channel.id)) return;
```